draft-thomson-http-scd.md

```diff
@@ -323,7 +323,7 @@ secondary server, without losing integrity with respect to the content that is
 distributed.
 
 This design relies on integrity and confidentiality for the request and response
-made to the origin server.  These requests MUST be made using HTTP over TLS
+made to the origin primary server.  These requests MUST be made using HTTP over TLS
```

Comment:
I think that we can remove origin in a lot of these cases.  "origin primary server" is a little unwieldy.